modules/040-control-plane-manager/images/kube-apiserver/patches/1.21/ingress_validation.patch

```diff
@@ -1,6 +1,6 @@
 diff --git a/pkg/apis/networking/validation/validation.go b/pkg/apis/networking/validation/validation.go
---- pkg/apis/networking/validation/validation.go
-+++ pkg/apis/networking/validation/validation.go
+--- a/pkg/apis/networking/validation/validation.go
++++ b/pkg/apis/networking/validation/validation.go
```

Comment:
I don't understand this change

Reply:
To make it in a common way: patch it with -p1 flag

Reply:
This might be a problem because it leads to restarts of all control plane components.

Reply:
Agree. But it's really uncomfortable to have different behavior. Everywhere we have patches with -p1 and only here it's different. And on update and patch backport it's really painful

Reply:
Basic image is also changed. We will have a restart in any case. So we can also fix this